app/scripts/transaction-manager.js

```diff
@@ -41,6 +40,20 @@ module.exports = class TransactionManager extends EventEmitter {
     return this.networkStore.getState().network
   }
 
+  setupProviderAndEthQuery ({provider, blockTracker, ethQuery}) {
+    if (this.provider) {
+      delete this.provider
+      delete this.blockTracker
+      delete this.query
+      delete this.txProviderUtils
```

Comment:
`delete this.xyz` is the same as `this.xyz = undefined`
since you overwrite the values below, you can remove this whole section